lightdash/lightdash
# feat: add OTP flow for email verification

Closes: #4635 

Blocked by: #4648 

### Description:

- Adds OTP table to db
- Adds models manage OTPs
- New endpoint to trigger OTP emails

### To test

See `examples/api/verify-emails.http` to:

- login
- check your verification status
- trigger a new top
- check verification status again
- try and verify with a passcode
